draft-ietf-quic-transport.md

```diff
@@ -5021,9 +4983,9 @@ See {{iana-error-codes}} for details of registering new error codes.
 
 Application protocol error codes are 16-bit unsigned integers, but the
 management of application error codes are left to application protocols.
-Application protocol error codes are used for the RST_STREAM
-({{frame-rst-stream}}) and APPLICATION_CLOSE ({{frame-application-close}})
-frames.
+Application protocol error codes are used for the RST_STREAM frame
+({{frame-rst-stream}}) and the CONNECTION_CLOSE frame with a type of 0x03
+({{frame-connection-close}}) frames.
```

Comment:
"the NAME frame with a type of 0xXX" is a very clunky reference convention.  Flags.

Reply:
I don't think you can avoid this clunkiness with flags here. You'd end up saying something like "CONNECTION_CLOSE frame with the X flag" instead.